ketohub/items.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
```

Comment:
Let's hold off on `items.py`, `middlewares.py` and `pipelines.py` since we won't necessarily use them.

We know we'll use at least one spider class, so can we add a class that inherits from `scrapy.Spider` in the `ketohub/spiders` directory? Like one that just returns an empty list and then we can add a dummy unit test for it (e.g. `self.assertTrue(True)`) just so we can exercise Coveralls.